BookingsApi/BookingsApi.DAL/Migrations/20211213145853_AddCaseHearingTypesAndRolesForScssTribunal.cs

```diff
@@ -0,0 +1,101 @@
+﻿using BookingsApi.Contract.Enums;
+using BookingsApi.DAL.Helper;
+using BookingsApi.Domain.RefData;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class AddCaseHearingTypesAndRolesForScssTribunal : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
+                nameof(CaseType),
+                new[] { "Id", "Name" },
+                new object[,]
+                {
+                    { 40, "SSCS Tribunal" }
```

Comment:
currently in dev the last id is at 38, any reason why we are skipping over 39?

![image](https://user-images.githubusercontent.com/84961314/145982533-cb81fcc6-a94f-45c6-9d16-39df84db8db9.png)

Reply:
actually all the ids appear to be out of sync with dev, I havent been able to check other envs as I dont have the passwords, is this done intentionally?